examples/config.toml

```diff
@@ -17,11 +17,10 @@ bin = "contracts/KovanBridge.bin"
 [authorities]
 accounts = [
 	"0x006e27b6a72e1f34c626762f3c4761547aff1421",
-	"0x006e27b6a72e1f34c626762f3c4761547aff1421",
-	"0x006e27b6a72e1f34c626762f3c4761547aff1421"
 ]
-required_signatures = 2
+required_signatures = 1
```

Comment:
changed example `required_signatures` to 1, so the transactions are actually relayed by a single authority